goproxy.go

```diff
@@ -274,11 +274,9 @@ func (g *Goproxy) serveFetchDownload(rw http.ResponseWriter, req *http.Request,
 		responseError(rw, req, err, false)
 		return
 	}
-	defer func() {
-		info.Close()
-		mod.Close()
-		zip.Close()
-	}()
+	defer info.Close()
+	defer mod.Close()
+	defer zip.Close()
```

Comment:
LGTM. The individual defer statements ensure all three `Close()` calls execute even if one panics. Thanks for the fix! 👍